app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -329,6 +332,14 @@ ControllerContainer::SetEnabled(const int32_t aControllerIndex, const bool aEnab
   m.SetVisible(m.list[aControllerIndex], aEnabled);
 }
 
+void
+ControllerContainer::SetModelVisible(const int32_t aControllerIndex, const bool aVisible) {
+    if (!m.Contains(aControllerIndex))
+        return;
+
+    if (m.list[aControllerIndex].modelToggle)
+        m.list[aControllerIndex].modelToggle->ToggleAll(aVisible);
```

Comment:
The first condition, `m.Contains(aControllerIndex)` seems very common in all similar methods. Not sure why this would be different. I'm ok with asserting both though.

Reply:
Right, what I meant was, if that's a precondition of the method we should assert. Otherwise, we could be allowing some code paths that are not correct.

Reply:
Fixed in new version.